### PR TITLE
Issue #KN-239 feat: Knowlg player app for PDF

### PR DESCRIPTION
**Pull request by santoshilimi**

### CODE REVIEW

**Review by HarishGangula — CHANGES_REQUESTED** [review]
Fix the module name and refactor the code

File: projects/shared/src/lib/player/player.component.html [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<div style="height:100vh;width:100%;" *ngIf="playerConfig?.metadata?.mimeType === 'application/pdf'">
```
Comment (HarishGangula) [inline projects/shared/src/lib/player/player.component.html]: Use ngSwitch and no inline style allowed

File: knowlg-app/src/app/home/services/content.service.ts [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+import * as _ from 'lodash-es';
+import {data} from './data';
+@Injectable({
+  providedIn: 'root'
+})
+export class ContentService {
+  playerConfig = {};
+  constructor() { }
+  getContenstList(mimeType) {
+    if (mimeType === 'pdf') {
```
Comment (HarishGangula) [inline knowlg-app/src/app/home/services/content.service.ts]: this should be implemented like `data[mimeType]` not with if conditions

**Review by HarishGangula — CHANGES_REQUESTED** [review]

File: projects/shared/src/lib/player/player.component.html (hunk 1 above)
Comment (santoshilimi) [reply projects/shared/src/lib/player/player.component.html]: Done

File: knowlg-app/src/app/home/services/content.service.ts (hunk 2 above)
Comment (santoshilimi) [reply knowlg-app/src/app/home/services/content.service.ts]: Done

**Review by HarishGangula — APPROVED** [review]

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**